ci/ci.sh

```diff
@@ -8,11 +8,6 @@
 
 set -euf -o pipefail
 
-################
-# Build client #
-################
-RUST_BACKTRACE=1 cargo build
```

Comment:
I think the CI is executing this line which is failing with this PR changes (the `MBEDTLS_INCLUDE_DIR` variable must be defined).

Reply:
You'll need to change this line to include the variable, otherwise I think it's failing on the `clippy` call because it needs to build too

Reply:
Yes, my bad, I forgot about that!! 🤦